js/cart.js

```diff
@@ -0,0 +1,78 @@
+
+        var btn_plus_length = document.getElementsByClassName('btn-plus').length;
+        var item_count = 1 ;
+        
+        for(let i = 0 ; i<btn_plus_length ; i++)
+        {
+            document.getElementsByClassName('btn-plus')[i].addEventListener("click",(e)=>{
+                
+                // get the value of quantity
+                let qtt_value = document.getElementsByClassName("field-qtt")[i].getAttribute("value");
+
+               // add one on click (+)
+               item_count = parseInt(qtt_value) + 1;
+               qtt_value = item_count;
+ 
+                // catch the element carry the total price 
+                let total_price = document.getElementsByClassName("item-total-price")[i];
+
+                // get the price 
+                let item_price = parseInt(document.getElementsByClassName("item-price")[i].getAttribute("data-price"));
+                 
+                // calculate the total price and append it to the element
+                total_price.textContent =  "Total price : " +(item_price * qtt_value) + " $";
+
+                // set the total price attribute = total price
+                var total_price_attribute =parseInt(total_price.getAttribute("total-price"));
+                total_price_attribute  =  item_price * qtt_value;
```

Comment:
Great names and descriptive. Good job. But use camel case better on future projects not now.